resources/capabilitystatement-bulk-data.json

```diff
@@ -0,0 +1,117 @@
+{
+  "resourceType": "CapabilityStatement",
+  "id": "bulk-data",
+  "url": "http://hl7.org/fhir/us/bulkdata/CapabilityStatement/bulk-data",
```

Comment:
I'm still expecting this to look like `http://hl7.org/fhir/bulk-data/CapabilityStatement/bulk-data`

Reply:
updated here https://github.com/HL7/bulk-data/pull/8/files#diff-7ca1fcb8dc53e609e860ea10f6c5e443